loader/loader_linux.c

```diff
@@ -236,9 +236,9 @@ VkResult linux_read_sorted_physical_devices(struct loader_instance *inst, uint32
                                             struct loader_phys_dev_per_icd *icd_devices,
                                             struct loader_physical_device_term **sorted_device_term) {
     VkResult res = VK_SUCCESS;
-    bool is_vulkan_1_1 = false;
+    bool app_is_vulkan_1_1 = false;
     if (inst->app_api_major_version >= 1 && inst->app_api_minor_version >= 1) {
```

Comment:
This version check is wrong I think. Again, not a showstopper issue.
```c
if (inst->app_api_major_version > 1 || inst->app_api_major_version == 1 && inst->app_api_minor_version >= 1)
```
This really begs for a helper function. Version checks are tedious to get correct.

Reply:
Actually major > 1 || minor > 1 should work because if it's newer than 1.x it will pass first check, if it's 1.2 it will pass second.  I'll make the change.

Reply:
Except no, because then (0,1) would pass...